mmv1/templates/terraform/custom_flatten/database_migration_service_connection_profile_oracle_forward_ssh_password.go.erb

```diff
@@ -0,0 +1,17 @@
+<%# The license inside this block applies to this file.
+    # Copyright 2023 Google Inc.
+    # Licensed under the Apache License, Version 2.0 (the "License");
+    # you may not use this file except in compliance with the License.
+    # You may obtain a copy of the License at
+    #
+    #     http://www.apache.org/licenses/LICENSE-2.0
+    #
+    # Unless required by applicable law or agreed to in writing, software
+    # distributed under the License is distributed on an "AS IS" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    # See the License for the specific language governing permissions and
+    # limitations under the License.
+-%>
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
```

Comment:
what is the advantage of these flatten functions vs ignore_read?

Reply:
answering my own question: ignore_read is not supported on nested fields: https://github.com/hashicorp/terraform-provider-google/issues/12410